views/pwas/form.hbs

```diff
@@ -18,12 +18,10 @@
   </head>
   <body>
     {{> header}}
-
-    {{#if pwa.manifestUrl}}
-    <div class="section-title">Edit a PWA</div>
-    {{else}}
-    <div class="section-title">Submit a PWA</div>
-    {{/if}}
+    <div class="section-title">
+      <a href="/" class="back"><i class="fa fa-chevron-left" aria-hidden="true"></i></a>
+      <div id="subtitle">{{#if pwa.manifestUrl}} Edit a PWA {{else}} Submit a PWA {{/if}}</div>
```

Comment:
We don't have a `/edit`endpoint anymore.  The Edit a PWA text seems to be a leftover from that. In fact, it seems to introduce a bug: When a submission of a new PWA fails and the page is reloaded, the text changes to "Edit a PWA". We should just keep the "Submit a PWA" text.